WMF Framework/CacheProviderHelper.swift

```diff
@@ -13,12 +13,26 @@ final class CacheProviderHelper {
         let responseFileName = cacheKeyGenerator.uniqueFileNameForItemKey(itemKey, variant: variant)
         let responseHeaderFileName = cacheKeyGenerator.uniqueHeaderFileNameForItemKey(itemKey, variant: variant)
         
-        guard let responseData = FileManager.default.contents(atPath: CacheFileWriterHelper.fileURL(for: responseFileName).path),
-            let responseHeaderData = FileManager.default.contents(atPath: CacheFileWriterHelper.fileURL(for: responseHeaderFileName).path) else {
+        guard let responseData = FileManager.default.contents(atPath: CacheFileWriterHelper.fileURL(for: responseFileName).path) else {
             return nil
         }
         
-        //let mimeType = FileManager.default.getValueForExtendedFileAttributeNamed(WMFExtendedFileAttributeNameMIMEType, forFileAtPath: responseFileName)
+        //missing response header file could happen as a result of migrating saved articles for mobile-html response and images.
+        //artifically returning a header here in this case, though maybe better would be to create a response header file during migration.
```

Comment:
One advantage of that approach would be that you could use reading the mime type from the file as the check that the file exists before setting `isDownloaded == true`